src/media/js/site/containers/tos.js

```diff
@@ -4,6 +4,22 @@ import {bindActionCreators} from 'redux';
 import urlJoin from 'url-join';
 
 import {signTOS} from '../actions/tos';
+import {Page} from '../components/page';
+
+
+export const tosUrl = urlJoin(process.env.MKT_ROOT,
+                              'developers/terms/standalone');
+
+
+export class TOSIframe extends React.Component {
```

Comment:
put this in the dumb components folder, especially since it's re-used by another smart component